lib/config/massage.spec.ts

```diff
@@ -42,6 +42,18 @@ describe('config/massage', () => {
       expect(res).toMatchSnapshot();
       expect(res.packageRules).toHaveLength(3);
     });
+    it('massages packageRules matchUpdateTypes with wildcard', () => {
+      const config: RenovateConfig = {
+        packageRules: [
+          {
+            // @ts-expect-error - Wildcard is not defined in our types
+            matchUpdateTypes: ['*'],
```

Comment:
I ran the ``npm run prettier-fix && npm run markdown-lint-fix && npm run eslint-fix && npm run doc-fix`` command to fix everything and push changes according to the styles defined for this project, but somehow the automatic linters appended this comma. Not sure if this is wanted here (it's not a very common practice).

Let me know if this is an error so I fix this.

Reply:
Prettier actually switched the default "trailing comma" behavior in the `2.0` release [^docs], quote from their docs:

> ## Trailing commas
> 
> Default value changed from `none` to `es5` in _v2.0.0_
> 
> Print trailing commas wherever possible in multi-line comma-separated syntactic structures. (A single-line array, for example, never gets trailing commas.)
>
> Valid options:
>
> `"es5"` Trailing commas where valid in ES5 (objects, arrays, etc.). No trailing commas in type parameters in TypeScript.

Anyway, we've telling Prettier to use the `es5` style trailing commas via the `.prettierrc` config file:

https://github.com/renovatebot/renovate/blob/cc34a1b4e918b267d6e928b57b31d853fe15223c/.prettierrc#L2

So you can keep the style/formatting as-is. 😉 

[^docs]: https://prettier.io/docs/en/options.html#trailing-commas